src/Cardano/Wallet/BlockSyncer.hs

```diff
@@ -54,7 +54,8 @@ tickingFunction getNextBlocks action tickTime = go
       go (BlockHeadersConsumed headersConsumed) = do
           blocksDownloaded <- getNextBlocks
           let blocksToProcess =
-                  filter (checkIfAlreadyConsumed headersConsumed) (L.nub blocksDownloaded)
+                  filter (checkIfAlreadyConsumed headersConsumed)
+                      (L.nub blocksDownloaded)
```

Comment:
Tiny note: when going for "multi-lining" a function call, I find it more readable to put a single argument per line, and a line break after the function name as well. e.g.:

```hs
let blocksToProcess = filter
        (checkIfAlreadyConsumed headersConsumed)
        (L.nub blocksDownloaded)
```